teemaderegister-react/src/components/HeaderWrap.jsx

```diff
@@ -185,6 +185,9 @@ class HeaderWrap extends Component {
               <div className='login'>
                 <Link to='/login'>Sign in</Link>
               </div>}
+            <div className='login'>
+              <Link to='/sign_up'>Sign up &nbsp;</Link>
+            </div>
```

Comment:
I would rather keep the signup button only on login page, e.g something similar to this and remove from page header. 
<img width="646" alt="Screenshot 2021-01-13 at 23 26 52" src="https://user-images.githubusercontent.com/4492641/104512429-dfccf880-55f6-11eb-8561-6afa01411246.png">

Reply:
✓